apps/tutorial_servo_control/main.cpp

```diff
@@ -0,0 +1,68 @@
+#include "mbed.h"
+#include "CAN.h"
+#include "TutorialServo.h"
+#include "CANMsg.h"
+#include "hw_bridge.h"
+void controlServo();
+float canMessengerServo();
+
+
+
+//set pins 
+AnalogIn potVoltageIn(PA_0);
+PwmOut servoPwmOut(PA_1);
+//CAN setup
+CAN can(CAN_RX, CAN_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANMsg rxMsg;
+CANMsg txMsg;
+#define TX_ID 0x100
+#define RX_ID 0x101
+
+
+
+
+int main() {
+  float percent;
+ 
+  TutorialServo servo_can(PA_1);	//create servo object attached to pin PA_1
+  txMsg.id = TX_ID;                 //set message ID   
+  
+  
+  txMsg << percent;
+
+  
+  while (1)

```

Comment:
you're missing a brace. plase try compiling this. instrucions on how to compile apps are in the readme